### ISSUE TITLE
Multiline ternary chains don't instrument correctly

### ISSUE THREAD

**Issue by chapati23** [sc-forks/solidity-coverage#192]
### Description
Both `truffle compile` (using global truffle) as well as `yarn truffle compile` (using local truffle) work. However, the second `truffle compile` that's running inside `solidity-coverage` throws an error during compilation and I don't understand how this could happen because it seems I'm using the same version everywhere. Is it an issue with the instrumentation?

### Versions
macOS 10.13.3
node.js 9.4.0
truffle (global) 4.0.1 with solidity 0.4.18
truffle (local) 4.0.1 with solidity 0.4.18

### The Error

This error occurs during *the second* compilation, if I understand `solidity-coverage` correctly, this means _after_ instrumentation

```
smart-contracts/coverageEnv/contracts/Cov.sol:27:9: TypeError: Wrong argument count for function call: 2 arguments given but expected 1.
      : balances[_address]
        ^
Spanning multiple lines.

smart-contracts/coverageEnv/contracts/Cov.sol:25:81: TypeError: True expression's type tuple(tuple(),uint8) doesn't match false expression's type uint256.
uint256 _totalPerTokenUnclaimedConverted; (,_totalPerTokenUnclaimedConverted) = totalPerTokenPayout == 0
                                                                                ^
```

<img width="1404" alt="pasted_image_08_02_18__12_59" src="https://user-images.githubusercontent.com/117495/35971889-f2b1fb44-0ccf-11e8-9f18-1d99b128004f.png">

### How To Reproduce
Here's a minimum contract to reproduce:

```solidity
pragma solidity 0.4.18;

import "zeppelin-solidity/contracts/token/PausableToken.sol";

contract TestToken is PausableToken {
  uint256 public totalPerTokenPayout;
  mapping(address => uint256) public claimedPerTokenPayouts;

  function TestToken () public { }

  function currentPayout(address _address)
    public
    view
    returns (uint256)
  {
    uint256 _totalPerTokenUnclaimedConverted = totalPerTokenPayout == 0
      ? 0
      : balances[_address]
      .mul(totalPerTokenPayout.sub(claimedPerTokenPayouts[_address]))
      .div(1e18);

    return _totalPerTokenUnclaimedConverted;
  }
}
```

**Comment by cgewecke** [comment]
@chapati23 This is very likely an instrumentation bug. We try to capture branch data for ternary operators and you are using a relatively complex multiline / chained expression. If you have a chance, could you see if the error is resolved by restructuring the code in your example as a conventional `if/else` statement?

**Comment by chapati23** [comment]
Indeed. If I get rid of the ternary, the TypeError disappears.

**Comment by cgewecke** [comment]
Fixed for the nth time in #830
